lib/kitchen/driver/vra.rb

```diff
@@ -64,6 +65,10 @@ class Vra < Kitchen::Driver::Base # rubocop:disable Metrics/ClassLength
       default_config :use_dns, false
       default_config :dns_suffix, nil
 
+      deprecate_config_for :tenant, Util.outdent!("
+        In vRA8.x, the 'tenant' configuration is no longer required for the authentication,
+        instead, please make advantage of the domain configuration.".dup)
```

Comment:
In vRA 8.x, the 'tenant' configuration is no longer relevant for authentication.
        Please use the 'domain' configuration in its place.